==== triton/test/Triton/reproducer.mlir#3 - /google/src/cloud/joelwee/mlir_a506279e5c5d5668e66b0749c26a93d8d373931a_1718622342/triton/test/Triton/reproducer.mlir ====
# action=edit type=text
--- triton/test/Triton/reproducer.mlir	2024-05-14 17:02:29.000000000 -0700
+++ triton/test/Triton/reproducer.mlir	2024-06-17 04:07:35.000000000 -0700
@@ -9,7 +9,7 @@
 {-#
   external_resources: {
     mlir_reproducer: {
-      pipeline: "builtin.module(any(convert-scf-to-cf,convert-index-to-llvm{index-bitwidth=0},convert-triton-gpu-to-llvm{compute-capability=90},convert-nv-gpu-to-llvm,convert-arith-to-llvm{index-bitwidth=0},canonicalize{  max-iterations=10 max-num-rewrites=-1 region-simplify=true test-convergence=false top-down=true},cse,symbol-dce,enable-line-info))",
+      pipeline: "builtin.module(any(convert-scf-to-cf,convert-index-to-llvm{index-bitwidth=0},convert-triton-gpu-to-llvm{compute-capability=90},convert-nv-gpu-to-llvm,convert-arith-to-llvm{index-bitwidth=0},canonicalize{  max-iterations=10 max-num-rewrites=-1 region-simplify=aggressive test-convergence=false top-down=true},cse,symbol-dce,enable-line-info))",
       disable_threading: false,
       verify_each: false
     }
